ci/builders/mac_unopt.json

```diff
@@ -13,7 +13,8 @@
                 "debug",
                 "--unoptimized",
                 "--no-lto",
-                "--prebuilt-dart-sdk"
+                "--prebuilt-dart-sdk",
+                "--impeller-enable-3d"
```

Comment:
The command to run tests for `host_debug_unopt` is defined down on line 167. Are there any additional tests to add there?

Reply:
as long as impeller_unittests runs it's fine - which it should as part of run_tests.py on this platform